tools/echo_server/echo_server.go

```diff
@@ -72,6 +72,17 @@ func secureEcho(certPath string, keyPath string, port string, verbose bool) {
 		MinVersion: tls.VersionTLS12,
 		RootCAs:    serverCAPool,
 		ClientAuth: tls.RequireAnyClientCert,
+		// Cipher suites supported by AWS IoT Core. Note this is the intersection of the set
+		// of cipher suites supported by GO and by AWS IoT Core.
```

Comment:
Would this be more accurate? -- "cipher suites supported by **OpenSSL** and by AWS IoT Core"

ps: https://www.openssl.org/docs/man1.1.0/man1/ciphers.html this is what I'm referring to.